src/main/java/io/spokestack/spokestack/tts/TTSManager.java

```diff
@@ -0,0 +1,304 @@
+    }
+
+    /**
+     * Get the current speech output component.
+     *
+     * @return The speech output component being managed by this subsystem.
+     */
+    public SpeechOutput getOutput() {
+        return output;
+    }
+
+    /**
+     * Construction only allowed via use of the builder.
+     *
+     * @param builder The builder used to construct this manager.
+     * @throws Exception if an error occurs during initialization.
+     */
+    private TTSManager(Builder builder) throws Exception {
+        this.ttsServiceClass = builder.ttsServiceClass;
+        this.outputClass = builder.outputClass;
+        this.config = builder.config;
+        this.context = builder.context.getApplicationContext();
+        this.lifecycle = builder.lifecycle;
+        this.listeners.addAll(builder.listeners);
+        prepare();
+    }
+
+    /**
+     * Synthesizes a piece of text or SSML, dispatching the result to any
+     * registered listeners.
+     *
+     * @param request The synthesis request data.
+     */
+    public void synthesize(SynthesisRequest request) {
+        this.ttsService.synthesize(request);
+    }
+
+    /**
+     * Registers the currently active lifecycle with the manager, allowing any
+     * output classes to handle media player components based on system
+     * lifecycle events.
+     *
+     * @param newLifecycle The current lifecycle.
+     */
+    public void registerLifecycle(@NonNull Lifecycle newLifecycle) {
+        Lifecycle currentLifecycle = this.lifecycle;
+        this.lifecycle = newLifecycle;
+        if (output != null) {
+            if (currentLifecycle != null) {
+                currentLifecycle.removeObserver(output);
+            }
+            this.lifecycle.addObserver(output);
+        }
+    }
+
+    @Override
+    public void close() {
+        release();
+        this.ttsService = null;
+        this.output = null;
+        this.listeners.clear();
+    }
+
+    /**
+     * Dynamically constructs TTS component classes, allocating any resources
+     * they control.
+     *
+     * @throws Exception If there is an error constructing TTS components.
+     */
+    public void prepare() throws Exception {
+        if (this.ttsService != null) {
+            throw new IllegalStateException("already prepared");
+        }
+
+        this.ttsService =
+              createComponent(this.ttsServiceClass, TTSService.class);
+        for (TTSListener listener : this.listeners) {
+            this.ttsService.addListener(listener);
+        }
+        if (this.outputClass != null && this.output == null) {
+            this.output = createComponent(this.outputClass, SpeechOutput.class);
+            this.ttsService.addListener(this.output);
+            this.output.setAppContext(this.context);
+            if (this.lifecycle != null) {
+                this.registerLifecycle(this.lifecycle);
+            }
+        }
+    }
+
+    private <T> T createComponent(String className, Class<T> clazz)
```

Comment:
`clazz` 
shameless

Reply:
The convention is shameless; I merely follow it.